### ISSUE TITLE
GUI: Show jaws moving

### ISSUE THREAD

**Issue by John-Holt-Tessella** [ISISComputingGroup/IBEX#3656]
As POLARIS instrument scientists I would like to see if my jaws are moving on the jaws page and the jaws manager page. This should be indicated by colour.
This should be extended to colouring all JAWs OPI.
Discuss whether there should be a readback which highlights or just the setpoint highlights.

**Comment by aaron-long** [comment]
https://github.com/ISISComputingGroup/ibex_gui/pull/815
